src/platforms/apple/common/usage.mdx

```diff
@@ -52,5 +52,8 @@ This integration will swizzle some methods to create breadcrumbs e.g.: for `UIAp
 
 ## Known Limitations
 
+- With 5.x.x, we added <PlatformLink to="/configuration/releases/">Release Health</PlatformLink>, which allows you to monitor your releases' health
+by observing user adoption, usage of the application, percentage of crashes, and session data. If you are using this feature, we recommend updating
+to the latest versions of 6.x.x because it contains many improvements to make the stats for Release Health more reliable.
```

Comment:
Another suggestion is to start with the important part:

Upgrade to 6.0.0 to get better RH data. And add the background story after that (when skimming just reads over)
```suggestion
- With 5.x.x, we added <PlatformLink to="/configuration/releases/">Release Health</PlatformLink>, which allows you to monitor your releases' health
by observing user adoption, usage of the application, percentage of crashes, and session data. If you are using this feature, we recommend updating
to the latest versions of 6.x.x because it contains many improvements to make the stats for Release Health more reliable.
```